content/02.main-text.md

```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
 | Digital Object Identifier (DOI) | DOI [Content Negotiation](https://citation.crosscite.org/docs.html) | `doi:10.1098/rsif.2017.0387` | [@doi:10.1098/rsif.2017.0387] |
 | PubMed Identifier (PMID) | NCBI [E-utilities](https://www.ncbi.nlm.nih.gov/books/NBK25501/) | `pmid:25851694` | [@pmid:25851694] |
 | PubMed Central Identifier (PMCID) | NCBI [Literature Citation Exporter](https://api.ncbi.nlm.nih.gov/lit/ctxp/) | `pmcid:PMC4719068` | [@pmcid:PMC4719068] |
-| arXiv identifier | [arXiv API](https://arxiv.org/help/api/index) | `arxiv:1502.04015v1` | [@arxiv:1502.04015v1] |
-| URL | [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1] | `url:https://lgatto.github.io/open-and-open/` | [@url:https://lgatto.github.io/open-and-open/] |
+| arXiv ID | [arXiv API](https://arxiv.org/help/api/index) | `arxiv:1502.04015v1` | [@arxiv:1502.04015v1] |
+| ISBN | [Zotero translation-server](https://github.com/zotero/translation-server) | `isbn:9780262517638` | [@isbn:9780262517638] |
```

Comment:
For the other identifiers we spell out the acronyms.  Should we use `International Standard Book Number (ISBN)`?

The example `@isbn:9780262517638` is okay, but if you want to reuse an existing book citation, we could use [Opening Science](https://link.springer.com/book/10.1007%2F978-3-319-00026-8#about) as the example here.  We currently cite it as `doi:10.1007/978-3-319-00026-8`.

Reply:
In https://github.com/greenelab/meta-review/pull/113/commits/8b9d518e2c12002a2153221d62a73f7fd4125eee, I cited the "Open Access Book" for the following sentence:

> For example, open access publishing provides an economic model that encourages maximal dissemination and reuse of scholarly articles